frontend/src/scenes/session-recordings/SessionRecordingsTable.tsx

```diff
@@ -227,7 +227,9 @@ export function SessionRecordingsTable({ personUUID, isPersonPage = false }: Ses
                         <DurationFilter
                             onChange={(newFilter) => {
                                 reportRecordingsListFilterAdded(SessionRecordingFilterType.Duration)
-                                setDurationFilter(newFilter)
+                                if (!isNaN(newFilter.value)) {
```

Comment:
I think we still want to set the filter in this case, just to a '0' value. Otherwise, if the user clears the text box, their old filter would still be applied.